polkadot/xcm/src/v5/mod.rs

```diff
@@ -493,13 +493,21 @@ pub enum Instruction<Call> {
 	///
 	/// - `origin_kind`: The means of expressing the message origin as a dispatch origin.
 	/// - `call`: The encoded transaction to be applied.
+	/// - `fallback_max_weight`: Used for compatibility with previous versions. Corresponds to the
+	///   `require_weight_at_most` parameter in previous versions. If you don't care about
+	///   compatibility you can just put `None`. WARNING: If you do, your XCM won't work with older
+	///   versions.
```

Comment:
We could still do best effort conversion - when `call` type is known we can get actual weight - see comment on the conversion code.

```suggestion
	///   compatibility you can just put `None`. WARNING: If you do, your XCM might not work with older
	///   versions. Make sure to dry-run and validate.
```